pytaxonkit.py

```diff
@@ -1112,10 +1113,17 @@ def lca(
         else:
             return None
     results = []
+    query_str_list = []
     for line in out.strip().split("\n"):
         queries, lcataxid = line.split("\t")
+        query_str_list.append(queries)
```

Comment:
Here, `queries` is a string containing the taxids of each individual LCA. E.g `"632 2461416"` is an LCA of the two taxids `632` and `2461416`. However not every query that was given to taxonkit is returned here. Queries that contain deleted taxids are just skipped entirely. This makes the number of queries given to taxonkit and the number of results returned to no longer be equal.